ffd_info_exchange/uscis/templates/confirmation-name-change.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1 class="headline-text">Thank you for applying for a name change!</h1>
+    <p>Thank you for submitting your name change application &mdash; your confirmation number is <strong>X7Ty51</strong>.</p>
+    <p>For status updates on your application, please call the USCIS customer service center at 1-800-666-5713.</p>
+
+    <div class="button_wrapper">
+        <!-- @todo: Get USWDS 'usa-button-small' class to apply. -->
```

Comment:
I'm having some trouble replicating this but I believe you're supposed to use the `usa-button` class too. so `<a class="usa-button usa-button-small">`